src/EntryPoints/Specials/SpecialApproverCategories.php

```diff
@@ -3,36 +3,31 @@
 namespace ProfessionalWiki\PageApprovals\EntryPoints\Specials;
 
 use MediaWiki\MediaWikiServices;
-use ProfessionalWiki\PageApprovals\Adapters\DatabaseApproverRepository;
+use ProfessionalWiki\PageApprovals\Application\ApproverRepository;
 use ProfessionalWiki\PageApprovals\Application\UseCases\GetApproversWithCategories;
+use ProfessionalWiki\PageApprovals\PageApprovals;
 use SpecialPage;
-use PermissionsError;
 use LightnCandy\LightnCandy;
 use WebRequest;
 
 class SpecialApproverCategories extends SpecialPage {
 
-	/**
-	 * @var DatabaseApproverRepository
-	 */
-	private $databaseApproverRepository;
+	private ApproverRepository $approverRepository;
```

Comment:
@waximabbax always use the PHP type, not a comment. And don't use the DB implementation as type